telemetry/vscode/src/generate.ts

```diff
@@ -81,10 +81,7 @@ function getArgsFromMetadata(m: MetadataType): string {
                 name,
                 kind: StructureKind.TypeAlias,
                 isExported: true,
-                type: m.allowedValues.map(v => {
-                    const modifiedVal = typeof v === "string" ? v.replace(/\s/g, "_") : v;
-                    return `'${modifiedVal}'`
-                }).join(' | '),
+                type: m.allowedValues.map(v => `'${v}'`).join(' | '),
```

Comment:
can a small be added for this https://github.com/aws/aws-toolkit-common/blob/main/telemetry/vscode/test/generator.test.ts

Reply:
Is it possible that allowedValues is undefined?